setuptools/lib2to3_ex.py

```diff
@@ -33,6 +34,13 @@ def run_2to3(self, files, doctests=False):
             return
         if not files:
             return
+
+        warnings.warn(
+            "2to3 support is deprecated. If the project still "
+            "requires Python 2 support, please migrate to "
+            "a single-codebase solution or employ an "
+            "independent conversion process.",
+            DeprecationWarning)
```

Comment:
Oh, actually, just noticed - don't we have our own deprecation warning that is visible by default?

I don't think too many people will see this.

Reply:
This one: https://github.com/pypa/setuptools/blob/master/setuptools/_deprecation_warning.py

Reply:
Good point - unless it's caught through a test runner than enables these warnings (or fails on them), this could be missed.

Reply:
Fixed in a354d7bc.